autoconnect/autoconnect-ws/autoconnect-ws-sm/src/identified/mod.rs

```diff
@@ -95,7 +95,7 @@ impl WebPushClient {
         broadcast_subs: BroadcastSubs,
         flags: ClientFlags,
         connected_at: u64,
-        current_timestamp: Option<u64>,
+        current_timestamp_us: Option<u64>,
```

Comment:
the unidentified code supplies this, it's `ms` there. The only `us` I see supplied is from a test, which is a bug in the test (and I think we probably have other similar incorrect timestamp usages in other tests)